### PR TITLE
update about-how-to.md

### PR DESCRIPTION
**Pull request by shreyas1599**
- Changed file name as per #55 
- Implemented changes suggested in #85
> This is the job of a How-to article.

Seems unnecessary
I also suggest rewording: 
>  How-to articles are often confused with tutorials, **but you won't fall into this trap** if you remember they are _problem_-oriented not _learning_-oriented.

To something like: 

> How-to articles are often confused with tutorials, which is not the case as they are _problem_-oriented not _learning_-oriented.

Will add a commit after review by a maintainer.

### CODE REVIEW

**Comment by jaredmorgs** [comment]
@shreyas1599 Yep I agree with your rewording of the explanation about how-to articles. Go ahead with your proposed fix to that paragraph.

**Comment by shreyas1599** [comment]
@jaredmorgs Done!

**Comment by shreyas1599** [comment]
@jaredmorgs can you close #55 and #85?
